server/utils/statemanager/prompt_creating_state.go

```diff
@@ -0,0 +1,29 @@
+package statemanager
+
+import (
+	"drawydraw/models"
+	"errors"
+)
+
+type promptCreatingState struct {
+	game *models.Game
+}
+
+func (state promptCreatingState) addPlayer(player *models.Player) error {
+	return errors.New("Add Player cannot be performed in the Prompt Creating State")
+}
+
+func (state promptCreatingState) startGame(groupName string, playerName string) error {
+	return errors.New("startGame not supported for initial prompt creation state")
+}
+
+func (state promptCreatingState) addPrompts(prompts *models.Prompts) error {
+	state.game.AddPrompts(prompts)
```

Comment:
we should check if AddPrompts errors out and return that error

Reply:
will do